build/build-image/Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 # This file creates a standard build environment for building Kubernetes
 
-FROM kube-build:cross
+FROM KUBE_BUILD_IMAGE_CROSS
```

Comment:
The better way was introduced in Docker 1.9 and involves the `ARG` statement (at least, I *think* you can use `ARG` with `FROM`), but I don't want to introduce a dep on Docker 1.9 yet.